### ISSUE TITLE
the solc@0.4.0 is not work!

### ISSUE THREAD

**Issue by zhouzhou84** [ethereum/solc-js#412]
Hello
I need to use solc@0.4.0 on my ubuntu16.04, so I use `npm install -g solc@0.4.0`,it install solc0.4.0 successfully , but when I use `solcjs --version`,it shows:
```
/home/dean/nodejs/lib/node_modules/solc/soljson.js:1
(function (exports, require, module, __filename, __dirname) { <!DOCTYPE html>
                                                              ^

SyntaxError: Unexpected token <
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:616:28)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/dean/nodejs/lib/node_modules/solc/index.js:3:26)
```
so I check the /home/dean/nodejs/lib/node_modules/solc/soljson.js, the content is like html-style:
```
<!DOCTYPE html>
<html>
  <head>
    <meta http-equiv="Content-type" content="text/html; charset=utf-8">
    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; style-src 'unsafe-inline'; img-src data:; connect-src 'self'">
    <title>Page not found &middot; GitHub Pages</title>
    <style type="text/css" media="screen">
      body {
        background-color: #f1f1f1;
        margin: 0;
        font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
      }
.......
```
Is there any solution to help me to compiler the solidity with solc@0.4.0?
thank you so much!

**Comment by chriseth** [comment]
Thanks for reporting this! We should fix this, but in the meantime, the recommended way to use old solidity versions is actually to use the latest npm package for solc-js and load an older version of the compiler itself. It is documented here: https://github.com/ethereum/solc-js#using-a-legacy-version

**Comment by zhouzhou84** [comment]
Thank you so much to notice this bug. I check the link and see the command `solc.loadRemoteVersion(version, callback)`, but to be honest I don't know how and where to use this command...So sorry...

**Comment by zhouzhou84** [comment]
If you fix the solc@0.4.0, could please give me a notice? Thank you very much!

**Comment by matheusaaguiar** [comment]
Closing for inactivity and problem was solved for later version.
